test/lender/Lender.sol

```diff
@@ -187,11 +187,11 @@ contract Lender {
   function lend(uint8 p, address u, uint256 m, uint256 a, uint256 mb, uint256 d) public returns (uint256) {
       // Instantiate market and tokens
       address market = IIlluminate(illuminate).markets(u, m)[p];
```

Comment:
don't use `market` here this way -- a market is an array of addresses to Illuminate, this is a specific principal or principal token... so the var name could be like `pToken` or `principal`... we use token quite a bit (i think we used `eToken` for this in the element method).. i'd say do whatever we are doing elsewhere

we can prob come back and standardize this particular lookup to always be `principal` -- that's what it is.